index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
Are you sure we should to do this?  I think a bootstrap max column width of 6 would work well.  Otherwise, it would span the entire page width.  Probably not desirable on a wide screen.  However, we can try it out and see.